docs/preview/index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Home"
+layout: default
+permalink: /
+redirect_from:
```

Comment:
This should be removed

Reply:
No, this is preview, will be moved instead of the index. So has to be present here.

Reply:
If we don't remove this we can't browse to it :) It's part of preview folder so don't think we need this.

Reply:
https://deploy-preview-134--arcus-security-staging.netlify.app/preview/index.html

Reply:
Yes, I actually think that's a good think that you can't browse to it. It's 'preview' as in 'not done'.

Reply:
I don't agree - Neither does the preview work nor people can't get an idea of what's coming (for example preview).

Don't see an issue in publishing it, they know it's experimental.

Reply:
Ok, I'll change this.

Reply:
Removed! Resolving.